crypto/obj/obj_dat.h

```diff
@@ -8832,7 +8844,7 @@ static const ASN1_OBJECT kObjects[NUM_NID] = {
     {"HKDF", "hkdf", NID_hkdf, 0, NULL, 0},
     {"KEM", "kem", NID_kem, 0, NULL, 0},
     {"KYBER512", "KYBER512", NID_KYBER512, 0, NULL, 0},
-    {"DILITHIUM3", "DILITHIUM3", NID_DILITHIUM3, 0, NULL, 0},
+    {"DILITHIUM3", "DILITHIUM3", NID_DILITHIUM3, 11, &kObjectData[6232], 0},
```

Comment:
What is 6232?

Reply:
`/* This file is generated by crypto/obj/objects.go. */`

This file is auto generated, but 6232 refers to the index of the structure  `kObjectData`, and the 6232 element is `/* NID_DILITHIUM3 */` the OID for Dilithium3